api/projects/exceptions.py

```
@@ -19,3 +19,40 @@ class TooManyIdentitiesError(APIException):
 class ProjectTooLargeError(APIException):
     status_code = 400
     default_detail = "Project is too large; Please contact support"
+
+
+from rest_framework import status
```

Ruff (0.16.0)

[error] Module level import not at top of file

(E402)